physical/order_by.go

```diff
@@ -68,7 +73,23 @@ func (node *OrderBy) Materialize(ctx context.Context, matCtx *MaterializationCon
 		return nil, errors.Wrap(err, "couldn't get execution node from order by source")
 	}
 
-	return execution.NewOrderBy(exprs, directions, sourceNode), nil
+	eventTimeField := octosql.NewVariableName("")
+
+	if node.orderByEventTime(node.Source.Metadata()) {
+		eventTimeField = node.Expressions[0].(*Variable).ExpressionName()
```

Comment:
This could be done better, as You need to parse `node.Expression[0]` twice.
Either delete that function and perform everything in this function or return `sourceMetadata.EventTimeField()` instead of boolean